app/models/manageiq/providers/vmware/cloud_manager/vm/reconfigure.rb

```diff
@@ -28,6 +28,15 @@ def disk_default_type
     'LSI Logic Parallel SCSI'
   end
 
+  def available_networks
+    ['None'] + orchestration_stack.cloud_networks.map(&:name)
```

Comment:
Not sure if `'None'` should not be added by the UI.

But if it is here, it needs to be `_('None')` or `N_('None')` so that we have a hope of translating it.

Reply:
@himdel yeah I agree adding None here seems like the wrong place

Reply:
"None" is a special network name which we pass directly to vCloud API and vCloud understands it, so using `_('None')` won't work unless we perform some kind of reverse transformation prior passing to vCloud. 

My thinking to add "None" here was that other providers don't seem to support it so I wasn't sure how to do it on UI (another IF statement with another function here on provider e.g. `supports_none?`). We can also make vLan field optional, but that would affect how reconfigure works for other providers.

That being said, please let me know if you'd still prefer to do it on UI and I'll try to do it.

Reply:
Oh interesting, maybe make the network selection optional on the UI and set the network name to `'None'` if `options[:network].nil?`